src/Symfony/Bundle/FrameworkBundle/KernelBrowser.php

```diff
@@ -122,7 +122,15 @@ public function loginUser($user, string $firewallContext = 'main'): self
 
         $token = new TestBrowserToken($user->getRoles(), $user, $firewallContext);
         $token->setAuthenticated(true);
-        $session = $this->getContainer()->get('session');
+
+        $container = $this->getContainer();
+        $container->get('security.token_storage')->setToken($token);
```

Comment:
replaced by `security.untracked_token_storage` in b004f3bf3bb871f88e3bc28f93b52295d6bcb97d

Reply:
Oops sorry I totally forgot about the existence of this service. Thanks!